libraries/provision/install_sync_gateway.py

```diff
@@ -45,44 +45,54 @@ def __str__(self):
-                                                                    self._build_number,
-                                                                    platform_extension)
+            package_name = "couchbase-{}-{}_{}-{}{}.{}".format(installer,
+                                                               sg_type,
+                                                               self._version_number,
+                                                               self._build_number,
+                                                               architecture,
+                                                               platform_extension)
         else:
             base_url = "http://latestbuilds.service.couchbase.com/builds/releases/mobile/couchbase-sync-gateway/{}".format(self._version_number)
-            package_name = "couchbase-{}-{}_{}_x86_64.{}".format(installer,
-                                                                 sg_type,
-                                                                 self._version_number,
-                                                                 platform_extension)
+            package_name = "couchbase-{}-{}_{}{}.{}".format(installer,
+                                                            sg_type,
+                                                            self._version_number,
+                                                            architecture,
+                                                            platform_extension)
         if aws:
             base_url = "https://cbmobile-packages.s3.amazonaws.com"
-            package_name = "couchbase-{}-{}_{}-{}_x86_64.{}".format(installer,
-                                                                    sg_type,
-                                                                    self._version_number,
-                                                                    self._build_number,
-                                                                    platform_extension)
+            package_name = "couchbase-{}-{}_{}-{}{}.{}".format(installer,
+                                                               sg_type,
+                                                               self._version_number,
+                                                               self._build_number,
+                                                               architecture,
+                                                               platform_extension)
         return base_url, package_name
 
     def sync_gateway_base_url_and_package(self, sg_ce=False,
                                           sg_platform="centos",
                                           sg_installer_type="msi",
                                           sa_platform="centos",
                                           sa_installer_type="msi", aws=False):
+        architecture = "_x86_64"
```

Comment:
already has default value in the method definition, right?

Reply:
sorry, the default value was set in another method definition. forget about what I said.